src/utils/useTxMulti/useTxMulti.ts

```diff
@@ -30,6 +34,28 @@ export type TUseTxMulti = () => {
 export const useTxMulti: TUseTxMulti = () => {
   const [state, dispatch] = useReducer(TxMultiReducer, initialState, identity, 'TxMulti');
   const getState = () => state;
+  const { accounts } = useContext(StoreContext);
+  const { addNewTxToAccount } = useContext(AccountContext);
```

Comment:
Naming: `addNew` is an oxymore. Prefer `addTxToAccount`.

Reply:
Let's rename when we refactor `AccountProvider`

Reply:
Is part of 3508